ci/long_running_distributed_tests/workloads/pytorch_pbt_failure.py

```diff
@@ -93,7 +93,7 @@ def optimizer_creator(model, config):
         momentum=config.get("momentum", 0.9))
 
 
-ray.init(address="auto" if not args.smoke_test else None, log_to_driver=True)
+ray.init(address="auto" if not args.smoke_test else None, _log_to_driver=True)
```

Comment:
Do not change other test files.

Reply:
I didn't change it, it's outdated modification during merging master.